plugins/ssr/ssripats.ml

```diff
@@ -254,17 +254,19 @@ let rec ipat_tac1 future_ipats ipat : unit tactic =
 
   | IPatTac t -> t
 
-and ipat_tac pl : unit tactic =
+and ipat_tac names_used pl : unit tactic =
   match pl with
   | [] -> tclUNIT ()
+  | (IPatClear _ as p1) :: (IPatView _ as p2) :: rest ->
```

Comment:
I seriously dislike such an ad-hoc case. We probably have to do it as a quick fix (because the regression is serious), but I hope there's some short term plan to get rid of it.